extensions/db/src/test/java/org/mule/extension/db/integration/delete/DeleteMetadataTestCase.java

```diff
@@ -46,7 +47,7 @@ public void bulkDeleteOutputMetadata() throws Exception {
   public void deleteNoParametersInputMetadata() throws Exception {
     ParameterMetadataDescriptor parameters =
         getInputMetadata("deleteMetadata", "DELETE FROM PLANET WHERE name = 'Mars'");
-    assertThat(parameters.getType(), is(typeBuilder.nullType().build()));
+    assertThat(parameters.getType(), is(instanceOf(VoidType.class)));
```

Comment:
why asserting with a `instaceOf(class)` instead of using the MetadataType equals as we did before?